events.md

```diff
@@ -0,0 +1,130 @@
+# Events
```

Comment:
Titles need to be Setext H1 syntax I believe.
https://github.com/kubecost/docs/blob/main/markdown.md#titles